Output function for simulation results, issue #26

ioumu8 opened the pull request:
Before front end is built, we need a clear way to read the simulation results.
This function change the simulation results into pandas DataFrame to make it easier to read the results.

chlin-git reviewed (changes requested):
1. `results_skin.py` won't be used by anyone except `simulation.py`, and at least for the time being we don't expect a second function needed in the future. I think we don't need to add an independent file just for a function that only one client uses.
2. This PR now  has conflicts with origin/master. Please resolve those confliction.

ioumu8 committed:
Adjust the ways of output for simulation results

Fit the simulation results into pandas Data Frame.
Make it easier to read the simulation results on command lines.

ioumu8 reviewed (commented):
Output format for eval_results is not built.

chlin-git commented on simulation.py:
```diff
@@ -78,20 +79,42 @@ def extract_data(raw_data, data_list, price_dataset, region):
             pickle.dump(output_data, file)
     if not quiet_mode:
-        print(alpha_path, results, eval_results, sep='\n')
+        print(alpha_path, out_put_on_cmd(results, instruments),
```
I found that this will print out only heads and tails of the data frame. Is it possible to print out all of them?

chlin-git commented on simulation.py:
```diff
@@ -78,20 +79,42 @@ def extract_data(raw_data, data_list, price_dataset, region):
-def handle_result(
-        results, eval_results, alpha_path,
-        quiet_mode, output_path):
+def out_put_on_cmd(results, instruments):
```
Can this be convert_to_dataframe?  It only converts `results` to format `pandas.DataFrame` rather than outputs anything.

chlin-git commented on simulation.py:
```diff
@@ -140,15 +165,13 @@ def main():
         if not results_queue.empty():
-            path, results, eval_results = results_queue.get()
-            unordered_results[path] = (results, eval_results)
+            path, results, eval_results, instruments = results_queue.get()
```
I prefer put `instrument` in the second position because it is more "basic".

chlin-git reviewed (approved):
LGTM
